.github/workflows/ci-model-regression.yml

```diff
@@ -0,0 +1,587 @@
+# The docs: https://www.notion.so/rasa/The-CI-for-model-regression-tests-92af7185e08e4fb2a0c764770a8e9095
+name: CI - Model Regression
+
+on:
+  push:
+    branches:
+    - '[0-9]+.[0-9]+.x'
```

Comment:
why do we need this?

Reply:
e.g. this would exclude `master`

Reply:
We don't wanna run the tests on the master branch, only in a PR.

Reply:
Not even for tagged releases?

Reply:
Just making sure, I understand this. We still want to run weekly scheduled runs on master and minor branches(e.g. `1.10.x`). Does this allow that?

Reply:
@dakshvar22 Scheduled version runs on the master branch and only the master branch - GH limitation

Reply:
@tczekajlo Ahh right, I now remember you telling this. Thanks for clarifying again. :)

Reply:
> Not even for tagged releases?

@wochinge I'll add tagged releases as well :)